setup.py

```diff
@@ -6,7 +6,7 @@
 
 setup(
     name="Open edX",
-    version="0.3",
+    version="0.3.1",
```

Comment:
You made this version change, @andy-armstrong. I'm ambivalent to whether or not we bump the version. I would think thought that we at least shouldn't bump the version until Teams is usable. So I will back out the version change, unless you and @dianakhuang disagree.